frontend/src/scenes/plugins/pluginsLogic.ts

```diff
@@ -110,11 +111,13 @@ export const pluginsLogic = kea<
                     capturePluginEvent(`plugin source edited`, response)
                     return { ...plugins, [id]: response }
                 },
-                updatePlugin: async ({ id }) => {
+                updatePlugin: async ({ id, pluginChanges }) => {
                     const { plugins } = values
-                    const response = await api.update(`api/organizations/@current/plugins/${id}`, {})
+                    const response = await api.update(`api/organizations/@current/plugins/${id}`, pluginChanges)
                     capturePluginEvent(`plugin updated`, response)
-                    actions.pluginUpdated(id)
+                    if (!Object.keys(pluginChanges).length) {
+                        actions.pluginUpdated(id)
+                    }
```

Comment:
I think it should somehow be more clear that if you call this function without pluginChanges, it will update the plugin to the latest available version... 🤔

Reply:
Yes, TBH I'd just move plugin **version** update to a separate POST-only action endpoint, like: `.../plugins/:id/upgrade`. WDYT?

Reply:
There's actually one edge case with the current update that I already bumped into.

I might see this:

![image](https://user-images.githubusercontent.com/53387/110039888-3080e800-7d42-11eb-8298-4b8f30b6f061.png)

... but by the time I'm done checking the diff and click "update", a new version of the plugin is out. I won't upgrade to what's on the screen but something else. If I meticulously scan the code of a plugin between version upgrades, then I might miss something.

What actually happens is that the "latest version" tag will remain what it says, but "current version" (`tag`) will update to the next version... and we'll still see that there's a new version available.

Passing a full `url` with the right tag to the update command would solve this... and make it all easier to reason about.